lua/lspconfig/configs/nextflow_ls.lua

```diff
@@ -0,0 +1,37 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'java', '-jar', 'nextflow-language-server-all.jar' },
+    filetypes = { 'nextflow' },
+    root_dir = function(fname)
+      return util.root_pattern('nextflow.config')(fname) or util.find_git_ancestor(fname)
+    end,
+    settings = {
+      nextflow = {
+        files = {
+          exclude = { '.git', '.nf-test', 'work' },
+        },
+      },
+    },
+  },
+  docs = {
+    description = [[
+https://github.com/nextflow-io/language-server.git
```

Comment:
```suggestion
https://github.com/nextflow-io/language-server
```

Reply:
plese check our new rule of adding new server https://github.com/neovim/nvim-lspconfig/blob/master/CONTRIBUTING.md#adding-a-server-to-lspconfig 
It seems that this project has not received much attention yet

Reply:
Yeah totally understand, the main reason for mentioning it's official from the programming language was to hope that could vet the validity of the language server early, especially since having it easily integrated into editors and adding that into the documentation on the editor modernization can help drive users rather than vice versa. I can close this and wait until it reaches the star count.

One question I do have is the VS code plugin for the language server will definitely get a lot more attention in the community early on and will get most of the stars just because that's what users who use this project will mainly be referencing. Would those stars help count towards this vetting process?

Reply:
number of vscode downloads can also be used as a reference.

Reply:
Oh that's awesome! The official VS code extension for the language that includes the language server has over 27k downloads: https://marketplace.visualstudio.com/items?itemName=nextflow.nextflow

Reply:
cool awesome. enough to add